library/test/src/test_result.rs

```diff
@@ -7,11 +11,16 @@ use super::types::TestDesc;
 
 pub use self::TestResult::*;
 
-// Return codes for secondary process.
+// Return code for secondary process.
 // Start somewhere other than 0 so we know the return code means what we think
 // it means.
 pub const TR_OK: i32 = 50;
-pub const TR_FAILED: i32 = 51;
+
+#[cfg(unix)]
+const SIGABRT: i32 = 6;
+
+#[cfg(windows)]
+const EXIT_ABORTED: i32 = -1073740791;
```

Comment:
Where does this name and value come from? Windows status codes are usually written in hex, so that's 0xC0000409, which appears to be `STATUS_STACK_BUFFER_OVERRUN`.

Ok, a little more digging reveals that our Windows [`abort_internal`](https://github.com/rust-lang/rust/blob/cb4d9a1902b3ea17e93872dafb76d24aa6295c47/library/std/src/sys/pal/windows/mod.rs#L323-L347) uses [`__fastfail`](https://learn.microsoft.com/en-us/cpp/intrinsics/fastfail?view=msvc-170), which is indeed documented to use code 0xC0000409.

Reply:
Updated to use hex here.